channels/views_test.py

```diff
@@ -174,7 +174,9 @@ def test_patch_field_channel_image(client, field_channel, attribute):
     png_file = os.path.join(  # noqa: PTH118
         os.path.dirname(__file__),  # noqa: PTH120
         "..",
-        "static",
+        "frontends",
```

Comment:
Does this path change affect the Django app elsewhere? I'm not clear on what it's testing.

Reply:
This particular change is fine.